src/main/java/kanbangantt/project/controller/helloController.java

```diff
@@ -4,12 +4,10 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
```

Comment:
```suggestion
@RestController
```

RestController 로 해도 충분할 것 같습니다.